.github/workflows/tests.yml

```diff
@@ -51,3 +51,9 @@ jobs:
             ${{ runner.os }}-go-
       - name: Run Tests
         run: make test
+      - name: Run Updater Integration Tests (non-linux)
+        if: matrix.os != 'ubuntu-20.04'
+        run: make test-updater-integration
```

Comment:
Assuming this works for windows. For mac we don't need `root` permissions?

Reply:
Nope, the tests on mac can be run rootless.